api/all.yaml

```diff
@@ -1906,6 +1951,9 @@ components:
         comment:
           description: User-provided free-form text description.
           type: string
+        owner:
+          description: The user that owns the model
+          type: string
```

Comment:
Why aren't there `created_by` and `updated_by` for models and model versions?

Reply:
I had already added them previously.